example.env

```diff
@@ -2,3 +2,5 @@
 
 BLOCKCHAIN_PROVIDER_URL=
 REGISTRY_CONTRACT_ADDRESS=
+API_BASE_URL=http://localhost:3003
+API_KEY=xxx
```

Comment:
Do you have any suggestions? I am not aware of a way to store configurations but storing them in a file

Reply:
usually npm packages exposes some `config()` method where you can pass eg. key. or it can be pass in constructor for object.

Reply:
the goal here is not to use `.env` at all, because this is just a package, not a service. I did that this way only because I want to do it fast, but this is not the best way.

Reply:
Don't we use .env just for our integration tests? Other users can pass configs directly to APIClient constructor

Reply:
oh.. its been a while and I forgot - looks like you right, ok lets keep it only for tests :) lets not use any env variables for providing functionality 👍